### ISSUE TITLE
Lox Hide Knappsack from Lox Cape loses Frost resist

### ISSUE THREAD

**Issue by kastleghost** [Vapok/AdventureBackpacks#123]
**Describe the bug**
When you craft the Lox Cape.  it offers Frost resist similar to Wolf Cape.  in vanilla Valheim people may have preferred this one as it was cheaper and in some cases looked better overall on other gear.  However when you get/craft the Lox Backpack it no longer has Frost Resist.

**To Reproduce**
Craft Lox cape.
Upgrade to Backpack.
See the loss

**Expected behavior**
It should transfer the same bonuses as the Cape.

**Screenshots**
![image](https://github.com/Vapok/AdventureBackpacks/assets/49766297/56f8fd41-671b-4b55-ba90-601b7571e2a1)

**Mod List (please list your mods that are being used):**
 - Adventure Backpacks 1.7.5

**Additional context**
Basically without having Frost Resist this is a pure downgrade, and in my opinion, not worth crafting/pursuing.

**Comment by Karmacopper** [comment]
This backpack is being rework soon from what I can gather on Vapok's discord.

**Comment by crispbcmd** [comment]
oh that is awesome, thanks!!

**Comment by Yanis01** [comment]
Hi, thanks for this mod, I really love it!
I would like to know if this is going to be fixed any time soon
I am currently playing with the lox backpack and the frost resistant is still off
I checked the settings but it does not appear in the Lox Backpack settings

**Comment by Vapok** [comment]
Okay, I found the issue. Bug introduced. This will be fixed in version 1.7.9.
